backend/app/routes/teamMember/deleteTeamMember.js

```diff
@@ -1,19 +1,33 @@
 const teamMemberModel = require('../../models/TeamMember');
-module.exports = async(req,res,next) => {
-    try {
-        const payload = res.locals.decode;
-        const memberId = req.body.memberId;
-        if (payload.isSuperAdmin === false) {
-            return res.status(401).json({ error: 'You are not an admin' });
-        }
-        const result = await teamMemberModel.findByIdAndDelete(memberId);
-        if(!result) {
-            return res.status(401).json({error:"Invalid id"});
-        }
-        return res.json({message:"Deleted successfully"});
+const fs = require('fs');
+const path = require('path');
+
+module.exports = async (req, res, next) => {
+  try {
+    const payload = res.locals.decode;
+    const memberId = req.body.memberId;
+
+    if (payload.isSuperAdmin === false) {
+      return res.status(401).json({ error: 'You are not an admin' });
     }
-    catch(error) {
-        return res.status(500).json({error:"Some internal server error"});
+
+    const member = await teamMemberModel.findById(memberId);
+    if (!member) {
+      return res.status(401).json({ error: 'Invalid id' });
     }
-    
-}
+
+    const result = await teamMemberModel.findByIdAndDelete(memberId);
```

Comment:
This result should have the detail of delete object, can't we do like below? - 
```
const result = await teamMemberModel.findByIdAndDelete(memberId);
if (!result) {
  return res.status(401).json({ error: 'Invalid id' });
}

const fileName = path.basename(result.image);
const imagePath = path.join(__dirname, '../../../uploads/teamMembersProfile/', fileName);
fs.unlink(imagePath, (err) => {
  if (err) {
    console.error('Failed to delete image file:', err);
  }
  console.log('Image file deleted:', imagePath);
});

return res.status(200).json({ message: 'Team member deleted successfully' });
```

Reply:
@Kajol-Kumari thank you. I forget that delete has the deleted object. so I make extra. Now i updated it.